Package.swift

```diff
@@ -0,0 +1,9 @@
+import PackageDescription
+
+let package = Package(
+    name: "GRDB.swift",
```

Comment:
I don't know if it affects anything at all, it doesn't seem to affect filenames in build dir nor import name. Can be changed to GRDB probably.
Module name is derived from directory name (it's currently 'GRDB', so it's 'import GRDB').